guava/src/com/google/common/cache/LongAdder.java

```diff
@@ -157,26 +159,30 @@ public String toString() {
    * @return the sum
    */
   @Override
-  public long longValue() {
-    return sum();
+  @SuppressWarnings("cast.unsafe")
```

Comment:
This method is documented as "equivalent to `sum`", but its annotations differ.  Shouldn't they be the same?

Reply:
LongAdder extends Striped64 which extends java.lang.Number.  Our annotated JDK has all the Value methods declared PolySigned so the overrides in LongAdder have to match to avoid warnings.